src/components/Canvas.vue

```diff
@@ -78,16 +89,36 @@ export default {
         keyUp(event) {
             this.keyEvent(event, false);
         },
+        newLinePush(x, y) {
+            this.lineList.push({
+                points: [x, y],
+                stroke: this.lineConfig.color,
+                strokeWidth: this.lineConfig.weight,
+            });
+        },
         draw() {
-            let lastPoint = [this.pointer.x, this.pointer.y];
+            let lastPoint = { x: this.pointer.x, y: this.pointer.y };
             if (this.direction['isUp']) this.pointer.y -= velocityOfPointer;
             if (this.direction['isDown']) this.pointer.y += velocityOfPointer;
             if (this.direction['isRight']) this.pointer.x += velocityOfPointer;
             if (this.direction['isLeft']) this.pointer.x -= velocityOfPointer;
-            if (lastPoint[0] != this.pointer.x || lastPoint[1] != this.pointer.y) {
-                this.lineConfig.points.push(this.pointer.x, this.pointer.y);
+            if (lastPoint.x != this.pointer.x || lastPoint.y != this.pointer.y) {
+                if (this.lineConfig.newLineFlag) {
+                    this.newLinePush(this.pointer.x, this.pointer.y);
+                    console.log(this.lineList);
+                }
+                this.lineConfig.newLineFlag = false;
+                this.lineList[this.lineList.length - 1].points.push(this.pointer.x, this.pointer.y);
+            } else {
+                this.lineConfig.newLineFlag = true;
             }
         },
+        movePointer(event) {
+            let stage = event.target.getStage();
+            let clickPos = stage.getPointerPosition();
+            this.pointer.x = clickPos.x;
+            this.pointer.y = clickPos.y;
+        },
```

Comment:
pointerを移動させた際にもlineを切らなきゃいけないので、応急処置ですが、ここにフラグを入れることでうまく行きました。
```suggestion
        movePointer(event) {
            let stage = event.target.getStage();
            let clickPos = stage.getPointerPosition();
            this.pointer.x = clickPos.x;
            this.pointer.y = clickPos.y;
            this.lineConfig.newLineFlag = true;
        },
```